app-i18n/transifex-client/transifex-client-0.14.2-r1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="A command line interface for Transifex"
+HOMEPAGE="https://pypi.org/project/transifex-client/ https://www.transifex.net/"
+SRC_URI="mirror://pypi/t/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-python/GitPython[${PYTHON_USEDEP}]
+	<dev-python/python-slugify-5.0.0[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	<dev-python/six-2.0.0[${PYTHON_USEDEP}]
+	dev-python/urllib3[${PYTHON_USEDEP}]"
+
+distutils_enable_tests setup.py
+
+src_prepare() {
+	eapply_user
+	sed -i -e 's:test_fetch_timestamp_from_git_tree:_&:' \
+		tests/test_utils.py || die
+	sed -i '/tests_require=["mock>=3.0.5,<4.0"]/d' setup.py || die
```

Comment:
```
>: cd /var/tmp/portage/app-i18n/transifex-client-0.14.2-r1/work/transifex-client-0.14.2/
>: grep -i mock setup.py 
    tests_require=["mock>=3.0.5,<4.0"],
>: sed -i '/tests_require=["mock>=3.0.5,<4.0"]/d' setup.py
>: grep -i mock setup.py 
    tests_require=["mock>=3.0.5,<4.0"],
>: sed -i '/tests_require=/d' setup.py
>: grep -i mock setup.py 
>: 
Seriously what is going on? Can someone explain? Is it **just** me?

Reply:
That sed doesn't work for me either, fine if brackets are escaped.
`sed -i '/tests_require=\["mock>=3.0.5,<4.0"\]/d' setup.py`

Reply:
Makes sense to escape the brackets, so I added that. Still wondering why it worked for me anyway...